cmd/object-handlers.go

```diff
@@ -3353,14 +3354,6 @@ func (api objectAPIHandlers) DeleteObjectHandler(w http.ResponseWriter, r *http.
 		return
 	}
 
-	if globalDNSConfig != nil {
-		_, err := globalDNSConfig.Get(bucket)
-		if err != nil && err != dns.ErrNotImplemented {
```

Comment:
It looks like we don't return NoSuchBucket here anymore ?

Reply:
We do not need to @vadmeste - since the request is always local and let the lower layers return appropriately. The lookup is not needed.

Reply:
The current check was never needed.